.ci/check_unsupported.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# Non-zero exit code is what we want to check
+set +e
+
```

Comment:
can we have `set -x` for debug output, its always helpful

Reply:
isn't this going to make a mess though?

Reply:
done below